snuba/datasets/schemas/tables.py

```diff
@@ -42,6 +51,7 @@ def __init__(self,
         *,
         local_table_name: str,
         dist_table_name: str,
+        mandatory_conditions: Sequence[Condition]=None,
```

Comment:
This won't/shouldn't type check, this needs `Optional`.

Reply:
You are right, This got a default in the last commit, which I made after looking at the --strict results of mypy. This has to be fixed.